src/cryptojwt/key_issuer.py

```diff
@@ -350,16 +351,18 @@ def __len__(self):
             nr += len(kb)
         return nr
 
-    def dump(self, exclude=None):
+    def dump(self, exclude=None, cutoff: Optional[list] = None) -> dict:
         """
         Returns the content as a dictionary.
 
+        :param exclude: Issuer that should not be include in the dump
```

Comment:
- `exclude_issuer` (perhaps make it `exclude_issuers` and a `Optional[List[str]]` to allow multiple issuers to be excluded?)
- `exclude_attrs` (or `exclude_attributes`)

Reply:
OK then.